torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -365,23 +376,20 @@ std::string ToXlaString(const c10::Device& device) {
 }
 
 const torch::lazy::BackendDevice* GetDefaultDevice() {
-  static std::string default_device_spec =
+  const std::string default_device_spec =
       UseVirtualDevice() ? "SPMD:0"
                          : runtime::GetComputationClient()->GetDefaultDevice();
   XLA_CHECK(!default_device_spec.empty());
-  static const torch::lazy::BackendDevice default_device =
-      ParseDeviceString(default_device_spec);
-  return &default_device;
+  g_default_device = ParseDeviceString(default_device_spec);
+  return &g_default_device;
 }
 
 c10::Device AtenDefaultDevice() {
   return XlaDeviceToAtenDevice(*GetDefaultDevice());
 }
 
 torch::lazy::BackendDevice GetCurrentDevice() {
-  if (!g_current_device) {
-    g_current_device = *GetDefaultDevice();
-  }
+  g_current_device = *GetDefaultDevice();
```

Comment:
If we're always overwriting `g_current_device`, wondering is it possible to rid of this global variable?